dev_envs/mac-env-setup.md

```diff
@@ -0,0 +1,260 @@
+# Setting up a Mac-based development environment #
+
+## Initial account setup (as administrator) ##
+
+For this section, log in as your administrator account.
+
+### Account permissions ###
+
+You’ll need to allow your standard user access to run `sudo` commands.
+First, you’ll want to take note of your account name by viewing the folders
+listed inside of the `/Users/` folder - in this example, the account
+name is **username**.
+
+```bash
+$ ls /Users/
+username        Shared        fedadmin
+```
+
+Then, you’ll add your **username** to the `/etc/sudoers` file using `visudo`:
+
+```sh
+$ sudo visudo
+...
+##
+# User specification
+##
+
+# root and users in group wheel can run anything on any machine as any user
+root        ALL = (ALL) ALL
+%admin      ALL = (ALL) ALL
+username    ALL = (ALL) ALL
+```
+
+Press `Esc` followed by `:wq` to save and quit `visudo`.
+
+### Install XCode command line tools ###
+
+Download XCode and Accept the License Agreement, then install the XCode
```

Comment:
`Accept the License Agreement` ➡️ `accept the license agreement`

Reply:
Resolved